ex_standards/ex_http_status.md

```diff
@@ -9,34 +9,34 @@ Some might ok with 2 options, choose one and justify it.
 ----
 
 * I cannot access that web if I'm not logged:
-  *  
+  *  403
 
 * I'm logged but I cannot access that page
-  * 
+  * 404
 
 * The item with that Id was deleted
-  * 
+  * 410
```

Comment:
410 é unha boa opción, pero normalmente a API non sabe que ese item existía, moi tipico en APIs REST HTTP polo que 404 é o caso máis habitual